requirements.txt

```diff
@@ -2,10 +2,10 @@ TA-Lib==0.4.17
 pyti==0.2.2
 pandas==0.24.2
 simplejson==3.16.0
-ccxt==1.18.523
+ccxt>=1.18.500
```

Comment:
why this?

Reply:
the precise ccxt version is not relevant for the usage here, which is limited to  - and this syntax will always take the "last available".

with the frequency of updates, we might risk to not be able to install freqtrade and technical (if both get their weekly update shited by a day) - since both require a different version.